app/assets/javascripts/admin/models/web-hook.js.es6

```diff
@@ -0,0 +1,98 @@
+import RestModel from 'discourse/models/rest';
+import { default as computed, observes } from 'ember-addons/ember-computed-decorators';
+
+export default RestModel.extend({
+  // a better way to sideload the list?
+  WILDCARD_EVENT: {
+    id: 1,
+    name: 'wildcard'
+  },
+  EVENTS: Em.A([{
+    id: 2,
+    name: 'topic'
+  }, {
+    id: 3,
+    name: 'post'
+  }, {
+    id: 4,
+    name: 'user'
+  }]),
+
+  contentTypes: Em.A([{
+    id: 1,
+    name: 'application/json'
+  }, {
+    id: 2,
+    name: 'application/x-www-form-urlencoded'
+  }]),
+  content_type: 1, // prefer json,
+  web_hook_event_types: Em.A([{ // default event type
+    id: 3,
+    name: 'post'
+  }]),
+  categoryFilters: Em.A(),
+
+  // FIXME: When enter event details from list, this observes are not used
+  // which results in an empty input box
+  @observes('category_ids')
+  updateCategoryFilters() {
+    this.set("categoryFilters", Discourse.Category.findByIds(this.get('category_ids')));
+  },
+
+  @computed('web_hook_event_types.[]')
+  existWildcardEvent() {
+    return this.get('web_hook_event_types').any(event => {
+      return event.name === 'wildcard';
```

Comment:
Instead of using the hardcoded string `'wildcard'`, prefer what you do in https://github.com/discourse/discourse/pull/4256/files#diff-c8e99453565042747d57cd4408ec83b3R63 instead